tools/compare_codegen.sh

```diff
@@ -110,7 +110,7 @@ cleanup() {
     git submodule update --init --recursive
 }
 
-trap "cleanup" INT TERM EXIT
+trap "cleanup" EXIT
```

Comment:
I was a little worried this would interfere with C-c but I checked and it seemed to still clean up, so it's all good.

Reply:
Yeah, I tested that and the PR way can still capture the ctrl-c signal.

It was the old way that keeps pushing down ctrl-c yields recursion disasters.